### DISCUSSION TITLE
Anonymize data, how ?

### DISCUSSION THREAD

**Discussion by StefanSa** [vectordotdev/vector#11525] [Q&A] [answered]
Hi there,
We have the requirement to anonymize incoming syslog data due to DSGVO / GDPR regulations.
Question, can Vector help here, if so how ?

**Comment by spencergilbert** [comment, marked as answer]
Hi @StefanSa - have you checkout out the [`redact`](https://vector.dev/docs/reference/vrl/functions/#redact) function yet?
